tests/unit/test_general_helper.py

```diff
@@ -28,18 +28,18 @@ def test_get_media_upload_path(movies):
                 assert os.path.isdir(media_upload_path)
```

Comment:
No theme is being added by Themerr for `TV Shows` or `TV Shows-tvdb`. Probably need to, either:

- disable Plex adding themes
- add option to overwrite plex themes (disabled by default)